src/config/get-config.js

```diff
@@ -43,6 +43,18 @@ module.exports = function getConfig() {
      * the log will not be generated.
      */
     LOG_MSG_CHANNEL: process.env.LOG_MSG_CHANNEL || 'moderation-activity',
+    /**
+     * The name of the role to suspend a user with.
+     * If not given, or if the role is not found
+     * the command will not function
+     */
+    SUSPEND_ROLE: process.env.SUSPEND_ROLE || `null`,
+    /**
+     * The name of the category to create new
+     * suspended user channels in.
+     * If not given, channel will not be created.
+     */
+    SUSPEND_CATEGORY: process.env.SUSPEND_CATEGORY || `null`,
```

Comment:
these can default to nothing rather than a string of `null` which is **nasty** as `null` is falsy, but a non empty string is **truthy**.

Reply:
Okay, I can adjust this. 🙂

Reply:
This is done.